### ISSUE TITLE
SingleFinder#find causes MySQL syntax error when using explicitly-specified multiple databases

### ISSUE THREAD

**Issue by JamesCropcho** [norman/friendly_id#69]
Hey Norman, I'm a big fan of friendly_id!

I have simultaneous connections to multiple databases configured. To allow associations/joins to work between models residing in different databases, I've subclassed ActiveRecord::Base (with .abstract_class = true), and added the following:

```
def self.table_name_prefix
    @XXX_data_prefix ||= "#{configurations['XXX_data']['database']}."
end
```

Makes everything except friendly_id work excellently.  When friendly_id executes a SELECT command, such as

```
SELECT * FROM `XXX_development`.`groups` WHERE (`XXX_development`.`groups.cached_slug` = 'XXX') AND ( (`XXX_development`.`groups`.`type` = 'RegularGroup' ) ) LIMIT 1
```

I get the error

```
Unknown column `XXX_development`.`groups.cached_slug` in 'WHERE' clause
```

I've forked friendly_id and believe i have fixed the problem here:

http://github.com/JamesCropcho/friendly_id/commit/765fb90b7b8f5626be0b37f59649ffd6d12c7ebc

Basically, doesn't look like you can pass a :conditions hash to AR::Base.find's clause which has a column name prepended by a database name and table name.

I compiled a gem from my fork's version, and am using it now without any issues (so far).

Let me know if there's anything I can do to help you resolve this.

James
